NetCoreDemos/Views/Toolbar/Multiline.cshtml

```diff
@@ -0,0 +1,337 @@
+
+@{
+    var toolbarModes = new [] { "Multiline mode", "Singleline mode" };
+    var listTypes = new [] {
+        new { icon = "orderedlist", alignment = "orderedlist", hint = "Ordered" },
+        new { icon = "bulletlist", alignment = "bulletlist", hint = "Bullet" }
+    };
+}
+
+<div>
+    @(Html.DevExtreme().Sortable()
+        .MinWidth(500)
+        .MinHeight(150)
+        .MaxHeight(370)
+        .Handles("right")
+        .Content(
+            Html.DevExtreme().Toolbar()
+                .ID("toolbar")
+                .Multiline(false)
+                .Items(items => {
+                    items
+                        .Add()
+                        .Widget(w => w
+                            .Button()
+                            .Icon("undo")
+                            .OnClick("undoButtonClick")
+                        )
+                        .Location(ToolbarItemLocation.Before);
+                    
+                    items
+                        .Add()
+                        .Widget(w => w
+                            .Button()
+                            .Icon("redo")
+                            .OnClick("redoButtonClick")
+                        )
+                        .Location(ToolbarItemLocation.Before);
+                    
+                    items
+                        .Add()
+                        .CssClass("toolbar-separator-container")
+                        .Template(new TemplateName("separator-template"))
+                        .MenuItemTemplate(new TemplateName("menu-separator-template"))
+                        .Location(ToolbarItemLocation.Before)
+                        .LocateInMenu(ToolbarItemLocateInMenuMode.Auto);
+
+                    items
+                        .Add()
+                        .Widget(w => w
+                            .DropDownButton()
+                            .DisplayExpr("Text")
+                            .KeyExpr("Size")
+                            .SelectedItemKey("14")
+                            .UseSelectMode(true)
+                            .DataSource(Model.FontSizes)
+                            .OnSelectionChanged("changeSize")
+                            .ItemTemplate(@<text><div style="font-size: <%- Text %>;"><%- Text %></div></text>)
+                        )
+                        .Location(ToolbarItemLocation.Before)
+                        .LocateInMenu(ToolbarItemLocateInMenuMode.Auto);
+
+                    items
+                        .Add()
+                        .Widget(w => w
+                            .DropDownButton()
+                            .DisplayExpr("Text")
+                            .KeyExpr("LineHeight")
+                            .SelectedItemKey("1.35")
+                            .StylingMode(ButtonStylingMode.Text)
+                            .UseSelectMode(true)
+                            .DataSource(Model.LineHeights)
```

Comment:
Why don't we use this [dataSource mapping](https://github.com/DevExpress/devextreme-demos/pull/2004/files#diff-5e7693e94b9ea43071c14047ac5aa50e14ecf1b638ed3b7e6dded4e4933215d6R378) in .net demos?

Reply:
We don't need it here)